app/scripts/common/components/oo-email-dialog.html

```diff
@@ -9,6 +9,12 @@ <h3 class="panel-title">
   <div class="panel-body">
     <form name="emailForm">
       <div class="row form-group">
+<div class="col-md-12">
+        <div class="alert alert-danger" role="alert" ng-show="!attachment && selectedMailTemplate.attachInvoice" translate>
+            Einige selektierte Einträge verfügen über kein angehängtes Rechnungsdokument. Diese E-Mails werden ohne Anhang verschickt.
```

Comment:
Sorry, I have to come back to this: Please try to add the 'translate' attribute to pure text elements. This one contains a <li .. tag and this will be included in the text to translate and this is ugly. Can you therefore add the text to a 'span'?